gem/templates/profiles/viewprofile.html

```diff
@@ -65,5 +65,13 @@
                 </form>
             </div>
         </div>
+        {% if user.gem_profile %}
+          <hr>
+          <div class="gender-form">
+              <div class="block">
+                      <h5>{% trans "Your gender is:" %}{{user.gem_profile.gender}}</h5>
```

Comment:
@rudigiesler I'm just not sure how to get the constant variables into the context here